### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Lileyy** [KotatsuApp/Kotatsu#1210]
### Brief summary

Hi, I just want to give some advice. Thank you in advance for making a really great application.I would like to suggest adding a disable nfsw comics feature in settings. So when I open a comic website(All websites whatever they are), all nfsw comics are not displayed. Thank you so much

### Kotatsu version

7.7.1

### Android version

10

### Acknowledgements

- [X] This is not a duplicate of an existing issue. Please look through the list of [open issues](https://github.com/KotatsuApp/Kotatsu/issues) before creating a new one.
- [X] This is not an issue with a specific manga source. Otherwise, you have to open an issue in the [parsers repository](https://github.com/KotatsuApp/kotatsu-parsers/issues/new/choose).

**Comment by Mhuzaifakhalid** [comment]
isn't this feature already available?

**Comment by Lileyy** [comment]
> isn't this feature already available?

Oh hello, can you tell me where?

**Comment by Mhuzaifakhalid** [comment]
settings> manga sources > disable NSFW

**Comment by Mhuzaifakhalid** [comment]
also, you can exclude NSFW from history:
Settings > Data and Privacy> Exclude NSFW from history

**Comment by Lileyy** [comment]
> settings> manga sources > disable NSFW

I've done that, but the adult comics are still there. What should I do? Even if I turn on the filter on the comics, they still reappear when I reopen them.

**Comment by Mhuzaifakhalid** [comment]
I think that is beyond the scope of the app, it depends on the source to tell it if it is NSFW or not. But I'm not sure, so please wait for a developer to respond

**Comment by Lileyy** [comment]
> I think that is beyond the scope of the app, it depends on the source to tell it if it is NSFW or not. But I'm not sure, so please wait for a developer to respond

Yes, maybe you can just add a feature to disable what genre in the settings, or turn off nfsw. Yes, I will wait for an answer from the developer. Thank you for answering my comment

**Comment by Koitharu** [comment]
This is a parser issue, I need a source name and a manga example

**Comment by Mhuzaifakhalid** [comment]
Mangafire. As in the screen recording, all of the manhwas get detected but no manga except "Kagurabachi"

I've tried many things, even reinstall etc but its persistent. Please solve this

**Comment by Lileyy** [comment]
> This is a parser issue, I need a source name and a manga example

For example, like batoto or manga sources that have nfsw. Thank you for answering me, I'm a little annoyed because it keeps popping up again every time I go to the source. Once again, thank you
